### PR TITLE
Document testing using opm command

### PR DESCRIPTION
**Pull request by gnufied**
Document how to use new index image format.

### CODE REVIEW

File: HACKING.md [hunk 1]
```diff
@@ -7,24 +7,50 @@ and then update `deploy/operator.yaml` to point to your images and follow rest o
...
+```
+
+If you are using podman then there is no need to specify container-tool option.
+
+8. Tag and push index image to quay.io (or a container registry of your choice). Make sure that images are publicly available.
+
+9. Create a catalogSource and subscribe to the source by applying `examples/olm/catalog-create-subscribe.yaml`.
+
+```
+~> oc create -f examples/olm/catalog-create-subscribe.yaml
```
Comment (jsafrane) [inline HACKING.md:50]: nit: I'd appreciate if there was explicit `vim examples/olm/catalog-create-subscribe.yaml` to replace the index image name with "your image"

**Comment by jsafrane** [comment]
Tested the new procedure, works for me.

File: HACKING.md (hunk 1 above)
Comment (gnufied) [reply HACKING.md:56]: done

**Pull request merged**